history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -1044,22 +1085,12 @@ you considers the substance of the Christian religion as a contingent truth;
 which contingent truth is matter of fact.
 From which you reason that:
 
-Matter of fact cannot be known but by the relation of another,
-or by perception of the outward senses;
-because there are naturally in our souls no ideas of contingent truths,
-such as are concerning necessary truths: namely, that God is;
-and that the whole is greater than the part.--And since it may without absurdity be said,
-that God cannot make a contingent truth to become a necessary truth;
-neither can God reveal contingent truths or matters of fact,
-but as contingent truths are revealed:
-but matters of fact are not revealed but by the outward senses.
-From which you concludes that
+[.syllogism]
```

Comment:
I dared to call this a syllogism, but not sure if it is because of how it's formatted...seems to fit its definition though, and a syllogism is mentioned later on in this text, perhaps referring to this? It wasn't clear to me where the syllogism was that was referred to.

Reply:
I like it, I think it fits.  Just tweak one thing: remove the blank lines between the syllogism parts., line 1090 and 1092